doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -1,103 +1,136 @@
-- For multicluster scenarios, you have installed Kuadrant on at least two different OpenShift clusters, and have a shared accessible Redis store.
-- You have the `kubectl` command line installed.
-- Optional: User workload monitoring is configured to remote write to a central storage system such as Thanos, as described in [Install Kuadrant on an OpenShift cluster](../../install/install-openshift.md).
+```bash
+kubectl create ns ${KUADRANT_GATEWAY_NS}
+```
 
-## Platform engineer workflow
+Create the secret credentials in the same namespace as the Gateway - these will be used to configure DNS:
 
-NOTE: You must perform the following steps in each cluster individually, unless specifically excluded. 
+```bash
+kubectl -n ${KUADRANT_GATEWAY_NS} create secret generic aws-credentials \
+  --type=kuadrant.io/aws \
+  --from-literal=AWS_ACCESS_KEY_ID=$KUADRANT_AWS_ACCESS_KEY_ID \
+  --from-literal=AWS_SECRET_ACCESS_KEY=$KUADRANT_AWS_SECRET_ACCESS_KEY
+```
 
-### Step 1 - Set your environment variables
+Before adding a TLS issuer, create the secret credentials in the cert-manager namespace:
 
-Set the following environment variables used for convenience in this guide:
 ```bash
-export zid=change-this-to-your-zone-id
-export rootDomain=example.com
-export gatewayNS=api-gateway
-export gatewayName=external
-export devNS=toystore
-export AWS_ACCESS_KEY_ID=xxxx
-export AWS_SECRET_ACCESS_KEY=xxxx
-export AWS_REGION=us-east-1
-export clusterIssuerName=lets-encrypt
-export EMAIL=foo@example.com
+kubectl -n cert-manager create secret generic aws-credentials \
+  --type=kuadrant.io/aws \
+  --from-literal=AWS_ACCESS_KEY_ID=$KUADRANT_AWS_ACCESS_KEY_ID \
+  --from-literal=AWS_SECRET_ACCESS_KEY=$KUADRANT_AWS_SECRET_ACCESS_KEY
 ```
 
-### Step 2 - Set up a DNS Provider
+### Deploy the Toystore app
+
+Create the namespace for the Toystore application:
 
-The DNS provider declares a credential to access the zone(s) that Kuadrant can use to set up DNS configuration. You should ensure that this credential only has access to the zones you want managed.
+```bash
 
-#### Create the DNS Provider Secret resource
+kubectl create ns ${KUADRANT_DEVELOPER_NS}
+```
 
-Apply the following `Secret` resource to each cluster. Alternatively, if you are adding an additional cluster, add it to the new cluster:
+Deploy the Toystore app to the developer namespace:
 
 ```bash
-kubectl create ns ${gatewayNS}
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${KUADRANT_DEVELOPER_NS}
 ```
 
-Create the secret credentials as follows:
+### Setup Toystore application HTTPRoute
 
+```bash
+
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: toystore
+  namespace: ${KUADRANT_DEVELOPER_NS}
+  labels:
+    deployment: toystore
+    service: toystore
+spec:
+  parentRefs:
```

Comment:
There is no parent ref here?

Reply:
Oh good catch added the rules parameter in twice